demisto_sdk/commands/unify/unifier.py

```diff
@@ -134,13 +141,7 @@ def merge_script_package_to_yml(self):
         """Merge the various components to create an output yml file
         """
         print("Merging package: {}".format(self.package_path))
-        package_dir_name = os.path.basename(self.package_path)
```

Comment:
why did we need to move this? why cant we just call this function from generate-docs ?